### ISSUE TITLE
Part Disign: Make the active body identifier more visible in tree view. (Feature request)

### ISSUE THREAD

**Issue by kaiwas** [FreeCAD/FreeCAD#17773]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I don't work with part design very often, but even rare attempts have revealed the inconvenience of identifying the active body in the tree view.
Bold text is, of course, good, but it is not enough. Because fonts and screen resolutions can be different. As a result, the current version is very poorly identified.

Even if you make the text underlined, the active body will be more visible.

But, probably, some special identifier is required. For example, it could be a contrasting frame around the text.

The concept is in the picture.

![image2](https://github.com/user-attachments/assets/f9741092-08e3-4f48-9da8-c0806546f2d3)

PS/ I have the overlay disabled. If you enable it, the current version is perceived even worse. Bold text is very dependent on the design theme.
![Снимок экрана_20241110_134525](https://github.com/user-attachments/assets/d6dbb161-d742-4de1-81e6-cd361af42b19)

### Full version info

```shell
OS: Manjaro Linux (KDE/plasma/xcb)
Architecture: x86_64
Version: 1.1.0dev.39141 (Git) AppImage
Build type: Release
Branch: main
Hash: 4bd80bdcbb97d6038bc7030aed9ecebf57f4e428
Python 3.11.10, Qt 5.15.15, Coin 4.0.3, Vtk 9.3.0, OCC 7.8.1
Locale: Russian/Russia (ru_RU)
Stylesheet/Theme/QtStyle: OpenLight.qss/OpenLight/Qt default
Installed mods: 
  * Curves 0.6.48
  * OpenTheme 2024.9.1
  * timber
  * PitchedRoof
  * CurvedShapes 1.0.10
  * FreeCAD_SketchArch
  * frame 0.1.1
  * lattice2 1.0.0
  * sheetmetal 0.4.26
```

### Subproject(s) affected?

None

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
This is a style sheet issue. OOTB with the classic theme the active Body is pretty clear.
![afbeelding](https://github.com/user-attachments/assets/9330f1a7-307e-42f7-acc9-53ce0a278b13)

**Comment by kaiwas** [comment]
> This is a style sheet issue. OOTB with the classic theme the active Body is pretty clear.

Yes. I mentioned that this parameter is heavily theme-dependent. Maybe there needs to be some way (Hardcode) to make the identification clear in 100% of cases?

**Comment by mwganson** [comment]
It is possible to both underline and overline the active body.  Tools menu -> edit parameters -> baseapp -> preferences -> tree view -> TreeActive???? various options
![Snip macro screenshot-94ddd9](https://github.com/user-attachments/assets/1ea7f655-4d25-449c-8568-a76b0067b68e)

You might need to create these boolean parameters if they do not already exist.  The color for the active object can be adjusted in edit menu - preferences -> display -> colors -> active container

**Comment by kaiwas** [comment]
> You might need to create these boolean parameters if they do not already exist.

Excellent! Thank you!
So the issue is resolved even more simply than expected. Is it enough to make the settings built into the program by default?
Maybe it is worth making these parameters available by default?

**Comment by jakerr** [comment]
With the classic theme, as mentioned above, the colors make it quite apparent, but even there I think there is room for improvement.

### Current state
When selection and active body are different elements in the tree it's very clear. The two tree elements are distinguishable by the colors set in `Preferences > Display > Colors > Tree View`
<img width="635" height="257" alt="Image" src="https://github.com/user-attachments/assets/5eb13e00-b7c9-4f82-afef-f48dd6ac383a" />
<img width="492" height="131" alt="Image" src="https://github.com/user-attachments/assets/fbdc4c29-8668-444c-b786-0fb7b2659597" />

However when the active body is also selected, it's harder to see. It is bold, but it's still hard to tell the difference.
<img width="496" height="151" alt="Image" src="https://github.com/user-attachments/assets/85649b0f-a193-4b79-8f0e-2c9e8e144899" />

### Suggestion

I think it would be nice to include the active body color on the selected item row as a tab like so, when the active body and selection coincide.
<img width="496" height="151" alt="Image" src="https://github.com/user-attachments/assets/509a5f7f-0e3e-4a37-8af0-37153cf574bf" />
